src/regtech_data_validator/create_schemas.py

```diff
@@ -104,7 +104,7 @@ def validate(schema: DataFrameSchema, submission_df: pd.DataFrame, max_errors: i
     is_valid = True
     findings_df: pd.DataFrame = pd.DataFrame()
     next_finding_no: int = 1
-    single_field = multi_field = register = 0
+    error_counts = warning_counts = 0
```

Comment:
these are `Count` instead of int now, right? so probably want to instantiate them as 0 counts?